gradle.properties

```diff
@@ -9,13 +9,13 @@
 #
 ### dependencies
 pravegaVersion=0.4.0-50.63aea88-SNAPSHOT
-flinkConnectorVersion=0.4.0-103.7971206-SNAPSHOT
+flinkConnectorVersion=0.4.0-100.d588197-SNAPSHOT
```

Comment:
This update is conflicting with the change in PR #147. Let's take into account in which order we merge them.